### PR TITLE
feat: add FlushTableExpr

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Introduce `FlashTableExpr`

## Checklist

- [ ]  I have written the necessary comments.
- [ ]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)
https://github.com/GreptimeTeam/greptimedb/issues/1087
https://github.com/GreptimeTeam/greptimedb/pull/1094

### CODE REVIEW

**Review by MichaelScofield — APPROVED** [review]

File: proto/greptime/v1/ddl.proto [hunk 1]
```diff
@@ -51,6 +51,12 @@ message DropTableExpr {
   string table_name = 3;
 }
 
+message FlashTableExpr {
```
Comment (v0y4g3r) [inline proto/greptime/v1/ddl.proto:54]:
```suggestion
message FlushTableExpr {
```

File: proto/greptime/v1/ddl.proto [hunk 2]
```diff
@@ -51,6 +51,12 @@ message DropTableExpr {
...
+  string catalog_name = 1;
+  string schema_name = 2;
+  string table_name = 3;
```
Comment (v0y4g3r) [inline proto/greptime/v1/ddl.proto:57]: Should we also allow user to flush a specific region with an optional `region_id` like clickhouse's `OPTIMIZE TABLE ... PARTITION ...` clause?

Comment (WenyXu) [reply proto/greptime/v1/ddl.proto:57]: > Should we also allow user to flush a specific region with an optional `region_id` like clickhouse's `OPTIMIZE TABLE ... PARTITION ...` clause?

Done👏

**Review by v0y4g3r — APPROVED** [review]
LGTM

**Pull request merged**